google-cloud-spanner/acceptance/spanner/client/params/pgjsonb_test.rb

```diff
@@ -0,0 +1,86 @@
+# Copyright 2022 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+require "spanner_helper"
+
+describe "Spanner Client", :params, :pgjsonb, :spanner do
+  let(:db) { spanner_pg_client }
+  let(:json_params) { { "venue" => "abc", "rating" => 10 } }
+  let :json_array_params do
+    3.times.map do |i|
+      { "venue" => "abc-#{i}", "rating" => 10 + i }
+    end
+  end
+
+  it "queries and returns a string parameter" do
+    skip if emulator_enabled?
```

Comment:
Totally optional, feel free to ignore (or argue that it's better for it to be explicit in every test):

This seems to be consistent across all tests. You could avoid repeating it in every test with:

```ruby
before do
  skip if emulator_enabled?
end
```

Reply:
Thanks Daniel, have updated accordingly 👍